Plugins/Reqnroll.Microsoft.Extensions.Logging.ReqnrollPlugin/build/Reqnroll.Microsoft.Extensions.Logging.props

```diff
@@ -0,0 +1,9 @@
+﻿<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="TaskFolder;TaskAssembly">
+  <ItemGroup>
+    <None Include="$(_Reqnroll_MicrosoftExtensionsLoggingPluginPath)" >
```

Comment:
Since this is not a real Reqnroll "plugin", but just a library, do we still need to copy the assembly like that?